testprojects/pants-plugins/tests/pants_test/new_package/test_pants_plugin_pants_requirement.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.base.build_environment import pants_version
+from pants.version import VERSION as _VERSION
+from pants_test.base_test import BaseTest
+
+
+class PantsPluginPantsRequirementTest(BaseTest):
```

Comment:
It's not clear that this is testing what it is testing... maybe an integration test that runs some pants command on the target?

Reply:
Thanks for pointing this out, this was completely removed.